helix-term/src/commands.rs

```diff
@@ -1454,7 +1454,8 @@ mod cmd {
             _ => bail!("invalid line ending"),
         };
 
-        doc_mut!(cx.editor).line_ending = line_ending;
+        let (view, doc) = current!(cx.editor);
+        doc.set_line_ending(view.id, line_ending);
```

Comment:
This should be done in a separate "convert" method. `set_line_ending` was supposed to be a way to reinterpret the current document without changing it.

\cc @cessen

Reply:
Ah, if it writes the line endings that way, then this change is probably unnecessary.

Reply:
Never mind, I thought that it would write to the file with the set line ending, but it doesn't seem to work that way. If it did, then `:reload` would be a good substitute.